.github/workflows/build.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         cd ./binaryen/build
         source $HOME/emsdk/emsdk_env.sh
         emcc --version
-        emcmake cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXE_LINKER_FLAGS="-sSTACK_SIZE=5242880"
+        emcmake cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXE_LINKER_FLAGS="-sMAXIMUM_MEMORY=4294967296"
```

Comment:
-sSTACK_SIZE=5242880 should be preserved. Also, not sure `-sMAXIMUM_MEMORY=4294967296` will work here. Did you check this?

Reply:
No, it doesn't need to be preserved, because of WebAssembly/binaryen#6135.

Also, that's where I added the flag (inside `-DCMAKE_EXE_LINKER_FLAGS`) when I built binaryen.js, so it should work.